benchmarks/bench_plot_randomized_svd.py

```diff
@@ -118,10 +118,7 @@
 
 
 def unpickle(file):
-    fo = open(file, 'rb')
-    dict = pickle.load(fo, encoding='latin1')
-    fo.close()
-    return dict['data']
+    return pickle.load(fo, encoding='latin1')["data"]
```

Comment:
Oups ;) Thanks it's now fixed.

2015-11-20 8:10 GMT+01:00 Raghav R V <notifications@github.com>:

> In benchmarks/bench_plot_randomized_svd.py
> <https://github.com/scikit-learn/scikit-learn/pull/5886#discussion_r45441045>
> :
>
> > @@ -118,10 +118,7 @@
> >
> >
> >  def unpickle(file):
> > -    fo = open(file, 'rb')
> > -    dict = pickle.load(fo, encoding='latin1')
> > -    fo.close()
> > -    return dict['data']
> > +    return pickle.load(fo, encoding='latin1')["data"]
>
> @sieben <https://github.com/sieben> You forgot the with open(file, 'rb')
> as fo: ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/5886/files#r45441045>.
>